validator/coordinator/src/main/java/tech/pegasys/teku/validator/coordinator/performance/DefaultPerformanceTracker.java

```diff
@@ -126,14 +126,16 @@ public void onSlot(UInt64 slot) {
       AttestationPerformance attestationPerformance =
           getAttestationPerformanceForEpoch(currentEpoch, analyzedEpoch);
 
-      if (mode.isLoggingEnabled()) {
-        statusLogger.performance(attestationPerformance.toString());
-      }
+      // suppress performance metric output when not relevant
+      if (attestationPerformance.numberOfExpectedAttestations > 0) {
```

Comment:
note: we should only suppress the logs - we do want to set the metrics to 0 if they're enabled.

Reply:
right, I just undid that after exploring further on how the metrics are used. added a simple test as well.